sdk/core/Azure.Core/samples/Events.md

```diff
@@ -0,0 +1,190 @@
+# Azure.Core Event samples
+
+**NOTE:** Samples in this file only apply to packages following the
+[Azure SDK Design Guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html).
+The names of these packages usually start with `Azure`.
+
+Most Azure client libraries offer both synchronous and asynchronous methods for
```

Comment:
```suggestion
Most Azure SDK .NET client libraries offer both synchronous and asynchronous methods for
```

Reply:
I added "for .NET" to match what we usually write like https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Blobs#azure-storage-blobs-client-library-for-net.